spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
-describe ManageIQ::Providers::IbmCloud::VPC::CloudManager::Refresher do
+# rubocop:disable Style/MethodCallWithArgsParentheses # Guidance does not conform to preferred expect formatting.
```

Comment:
This is already excluded by manageiq-style's rubocop base file and isn't necessary:
```
spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb:3:1: W: [Correctable] Lint/RedundantCopDisableDirective: Unnecessary disabling of Style/MethodCallWithArgsParentheses.
# rubocop:disable Style/MethodCallWithArgsParentheses # Guidance does not conform to preferred expect formatting.
```

Reply:
My local rubocop warns me of this, thus the reason I disable it. I also get warnings about `{key: "b"}` should be `{:key => 'b'}` which means that my local rubocop is using the manageiq style. Might want to check that the directive is working as expected.

Reply:
If I delete these `rubocop:disable` directives locally then rubocop doesn't warn about this, miq-bot also doesn't so it seems more likely an issue with your local rubocop.

Reply:
Dunno, it could be the way that VS Code calls rubocop messes with the paths. There have also been a number of fixes to this cop between the version run in miq-bot and the latest. Maybe miq-bot just doesn't see it because it is broken in that build.  In any case, when I am developing without these disables I get a ton of false positives. For me it is more important to have this work in VS Code than it is to have miq-bot pass.

Reply:
This is working in Atom, not sure about Rubymine (@Fryguy ?), so probably vscode.  I see a number of issues with the vscode plugin not honoring inherit_from directives (e.g. https://github.com/rubyide/vscode-ruby/issues/227#issuecomment-694394518)

Reply:
We shouldn't be adding these to the code base because an editor isn't honoring `.rubocop.yml` correctly.

Reply:
I use Sublime - I had to just tweak the rubocop plugin to `bundle exec rubocop` instead of straight `rubocop` so it uses the right one, but that works

Reply:
(actually to be honest, I think I just installed 0.82.0 and use that only, and didn't bother tweaking the plugin... our app will automatically install 0.82.0 via manageiq-style as a dependency, and I just intentionally don't install a newer version)

Reply:
Also see https://github.com/rubyide/vscode-ruby/issues/227#issuecomment-740521812

Reply:
And for atom you just have to drop
```
  "linter-rubocop":
    useBundler: true
```
in `.atom/config.cson` (or use the UI settings for the rubocop plugin).

I wonder if it'd be helpful to have this in the developer_setup guide

Reply:
I recall in the past, there was a weird issue with sublime where it launches a non login shell to run its plugin scripts, and therefore would use the system Ruby instead of my Ruby with my Ruby version manager.  The two options to fix were to either install the correct rubocop in the system Ruby _or_ change my zprofile to switch to the expected Ruby.  I did the latter and then never looked back, because that also changes out for all my login shells as well.

Reply:
The proposed workaround for VSCode doesn't seem to work. Considering the issue has been open since 2017, I doubt there will be a resolution anytime soon. So that leaves 2 options:
1. Add this cop to the rubocop_local with the spec exclude.
2. Leave the disables in the specs.

I did 1 already and had the change removed, thus the reason I'm doing 2.